src/input/target/xb360.rs

```diff
@@ -218,6 +219,19 @@ impl TargetInputDevice for XBox360Controller {
         }
         Some(self.queued_events.drain(..).collect())
     }
+
+    /// Clear any local state on the target device.
+    fn clear_state(&mut self) {
+        let caps = self.get_capabilities().unwrap_or_else(|_| {
+            log::error!("No target device capabilities found while clearing state.");
+            Vec::new()
+        });
+        for cap in caps {
+            let ev = NativeEvent::new(cap, InputValue::Bool(false));
```

Comment:
This might not work as expected for analog sticks. When any evdev-based target devices translate this `NativeEvent` to an `EvdevEvent`, it will use this logic:
https://github.com/ShadowBlip/InputPlumber/blob/main/src/input/event/evdev.rs#L885-L958

Since the value is set to `InputValue::Bool(false)` and there is axis information available, this will get translated as the minimum value of the axis, which would be like pressing the stick fully in one direction. We may want to consider either adding some logic to handle gamepad axes here, or potentially adding a new `InputValue::Zero` type that could be translated to the appropriate "zero" value for each kind of input.